### PR TITLE
Offlinewebserver - Allow the web server to be used in a closed-network

### PR DESCRIPTION
**Pull request by nieknooijens**
do remember that we now have a lot of web files, so I moved them to their own "www" folder.
since cmake only overwrites/updates existing files during "make install" you should remove the old index.html from your install folder (usually /usr/local/share/games/performous)

### CODE REVIEW

File: data/themes/default/www/index.html [hunk 1]
```diff
@@ -8,9 +8,9 @@
     <meta name="apple-mobile-web-app-status-bar-style" content="black">
     <title>Performous web frontend</title>
 
-    <link rel="stylesheet" href="http://code.jquery.com/mobile/1.3.2/jquery.mobile-1.3.2.min.css">
-    <script src="http://code.jquery.com/jquery-1.9.1.min.js"></script>
-    <script src="http://code.jquery.com/mobile/1.3.2/jquery.mobile-1.3.2.min.js"></script>
+    <link rel="stylesheet" href="jquery.mobile-1.3.2.min.css">
+    <script src="jquery-1.9.1.min.js"></script>
+    <script src="jquery.mobile-1.3.2.min.js"></script>
```
Comment (Baklap4) [inline data/themes/default/www/index.html]: Maybe update to the new jquery/jquerymobile?

Comment (nieknooijens) [inline data/themes/default/www/index.html]: that breaks the site and is not the purpose of this pull request. updating jquery mobile should be a seperate issue with a seperate pull-request

Comment (Baklap4) [inline data/themes/default/www/index.html]: Will test it out myself since i have to prep my laptop for animecon anyway ;)

**Comment by Baklap4** [comment]
Just tested it works in chrome and firefox so i think this can be merged :+1:

**Pull request merged**